lib/chef/licensing_config.rb

```diff
@@ -0,0 +1,18 @@
+require "chef-licensing"
+require_relative "log"
+
+class Chef
+  class LicensingConfig
+    INFRA_ENTITLEMENT_ID = "a5213d76-181f-4924-adba-4b7ed2b098b5".freeze
+    COMPLIANCE_ENTITLEMENT_ID = "3ff52c37-e41f-4f6c-ad4d-365192205968".freeze # InSpec's entitlement ID
```

Comment:
see magic strings comment above from Thomas

Reply:
This is not needed in this PR and removed it.